server/lib/resolvers/resolverMapper.test.js

```diff
@@ -11,30 +11,32 @@ const resolverMapper = require('./resolverMapper')
  */
 
 test('should create Postgres resolvers successfully with no Handlebars templates', t => {
-  const dataSourceDefs = {
-    'p1': {
-      'type': 'postgres',
-      'config': {
-        'not_really_important': 'not_an_integration_test'
-      }
+  const dataSourceDefs = [{
+    'type': 'Postgres',
+    'name': 'p1',
+    'config': {
+      'not_really_important': 'not_an_integration_test'
     }
-  }
+  }]
   const dataSources = dataSourceParser(dataSourceDefs, false) // do not connect!
 
-  const resolverMappings = {
-    'q1': {
-      'type': 'Query',
-      'dataSource': 'p1',
-      'requestMapping': 'q1_requestMapping {{ var }}',
-      'responseMapping': 'q1_responseMapping {{ toJSON "foo" }}'
+  const resolverMappings = [{
+    'type': 'Query',
+    'field': 'q1',
```

Comment:
@darahayes I wonder why Dave named this as 'field'? Isn't it basically the mappingKey or mappingId?

Reply:
@aliok I'm not really sure to be honest. It makes the code a bit harder to follow also. I would much prefer `mappingKey`, `mappingId` or even just `name`

Reply:
Do note that the database schema for resolvers mappings has an `id` column as well as a `field` column.